sandbox/cgroup/cgroup.go

```diff
@@ -61,7 +62,22 @@ var (
 )
 
 func init() {
+	dirs.AddRootDirCallback(func(root string) {
+		rootPath = root
+	})
 	probeVersion, probeErr = probeCgroupVersion()
+	if probeErr == nil {
+		pickVersionSpecificImpl()
+	}
+}
+
+func pickVersionSpecificImpl() {
+	switch probeVersion {
+	case V1:
+		freezerV1Impl()
```

Comment:
This is an extreme nitpick, but since this is a method that performs some action, it would be better named as a verb. Suggestions: `pickFreezerV1Impl()`, `selectFreezerV1Impl()`, etc.